lib/types.js

```diff
@@ -2071,6 +2072,7 @@ function RecordType(schema, opts) {
   }, this));
   this._branchConstructor = this._createBranchConstructor();
   this._isError = schema.type === 'error';
+  this._noStackTraces = opts.noStackTraces;
```

Comment:
should this happen in the `RecordType` constructor or in the `Type` constructor?